src/Microsoft.DotNet.Cli.Sln/FileManpulation/ByteOrderMark.cs

```diff
@@ -0,0 +1,121 @@
+// 
```

Comment:
This file seems to be copied from somewhere. Also I believe you should `git commit --author "Zlatko ...."`